metalearn/metafeatures/metafeatures.py

```diff
@@ -1,17 +1,12 @@
-import os
-import math
-import json
 import time
-import io
-import copy
+from copy import deepcopy
 from typing import Dict, List
 
 import numpy as np
-import pandas as pd
```

Comment:
same here. let's just import pandas as use `pd.Series` and `pd.DataFrame` everywhere.